src/log.h

```diff
@@ -243,6 +243,23 @@ namespace librealsense
                 //el::Helpers::uninstallLogDispatchCallback< el::base::DefaultLogDispatchCallback >( "DefaultLogDispatchCallback" );
             }
         }
+
+        //stop logging and reset logger to default configurations
+        void reset_logger()
+        {
+            el::Configurations defaultConf;
+            defaultConf.setToDefault();
```

Comment:
Would this not cause any other settings to be lost?

Reply:
done